lambdaclass/curse_of_myrra
# Players can choose their name

Closes #527 

A player can now choose their name when starting the game, and it will be kept until it's changed.

Things to test:

1. Start the game and make sure a pop up appears, asking you to enter your name.
2. Start a new game, and observe you're no longer called "Player 1", but the name you just picked. This applies to the lobby, the character selection scene, the match and the death splash.
3. Close the application and open it again. This time you shouldn't get the pop up, as your name was saved in the device you're using.
4. Use the button in the lobbies scene to change your name and verify it's working correctly.
5. Play a multiplayer match (remember to do it in your localhost) and check that the name assignments are being made correctly.
